snakePipes/workflows/RNA-seq/Snakefile

```diff
@@ -133,7 +133,7 @@ def run_alignment_free():
 
 
 def run_alignment():
-    if "alignment" in mode.split(","):
+    if "alignment" in mode:
```

Comment:
Thanks, I had already fixed that on our local copy but forgot to do so on `develop`!